app/models/party.rb

```diff
@@ -26,10 +26,10 @@ class Party < ApplicationRecord
 
   def standing_in(ons_id)
     if smv_code.to_sym == :plaid
-      return ons_id[0] == 'W'
+      return ons_id[0] == "W"
```

Comment:
Would be nice to squash these fixes into the originating commit within the PR in future, to keep the commit history clean, avoid tests failing on certain commits, and keep the possibility of using `git bisect` to find other issues.

Reply:
I try not to get to this situation, but I missed these locally (maybe they're missed by our rubocop/guard combo, unless you see it at the time of the actual commit), but these only surfaced after the PR was created.  So I tend to simply add commits to an existing PR just so it's clear what's happened, because I lean towards not doing anything that would need a force push after the PR was created, because it's not always obvious why a force-push has happened, and it can get messy if someone has already pulled the branch.

Next time I hit this I will try to fix similar things quickly after the PR as raised.

Reply:
No worries.  Personally [I prefer a force-push](https://blog.adamspiers.org/2017/08/16/squash-merging-and-other-problems-with-github/) because even though [GitHub](https://github.com/isaacs/github/issues/997) [miserably](https://github.com/isaacs/github/issues/998) [fails](https://github.com/isaacs/github/issues/999) to provide a good UI for this, it's just about usable enough.  But this is down to personal preference really.